friture/plotting/coordinateTransform.py

```diff
@@ -58,25 +58,31 @@ def setBorders(self, start, end):
     def setScale(self, scale):
         self.scale = scale
 
-    @pyqtSlot(float, result=float)
+    @pyqtSlot(float, result=int)
```

Comment:
if it's possible, I think this method should keep returning floats. It might sound strange that screen coordinates would not be integers, but with display scaling this might not be that unusual